docs/quick-tutorial.md

```diff
@@ -118,13 +120,13 @@ global:
 
 ## Querier/Query
 
-Now that we have setup the Sidecar for one or more Prometheus instances, we want to use Thanos' global [Query Layer](components/query.md) to evaluate PromQL queries against all instances at once.
+Now that we have setup the sidecar for one or more Prometheus instances, we want to use Thanos's global [Query Layer](components/query.md) to evaluate PromQL queries against all instances at once.
 
-The Query component is stateless and horizontally scalable and can be deployed with any number of replicas. Once connected to the Sidecars, it automatically detects which Prometheus servers need to be contacted for a given PromQL query.
+The Querier component is stateless and horizontally scalable, and can be deployed with any number of replicas. Once connected to the sidecars, it automatically detects which Prometheus servers need to be contacted for a given PromQL query.
```

Comment:
```suggestion
The Querier component is stateless and horizontally scalable, and can be deployed with any number of replicas. Once connected to the Sidecars, it automatically detects which Prometheus servers need to be contacted for a given PromQL query.
```

Reply:
Same thing with `the`, and since this is plural I think it's best to prepend a clarifying `Thanos` to `Sidecars`

Reply:
Actually the more I look at this the more I feel like I've forgotten how to English... since it's plural, keeping `the` may actually make the most sense. I don't know if there's a formal rule for that, but that way sounds more intuitively natural.